### PR TITLE
[Docs] Describe multiple browser windows mode limitations 

### LINKED ISSUE #5905: [Docs] Describe limitations in the multi-window mode (including video-related)
At this moment page in video can be stretched/narrowed if the windows size differ

### PR DESCRIPTION
**Pull request by DIRECTcut**
fixes #5905

### CODE REVIEW

**Review by miherlosev — APPROVED** [review]

File: docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md [hunk 1]
```diff
@@ -162,3 +162,19 @@ test('Close a specific window', async t => {
     await t.closeWindow(window1);
 });
 ```
+
+## Limitations
+
+This section describes the limitations of multiple browser windows testing in TestCafe.
+
+### Limited Video Recording
```
Comment (VasilyStrelyaev) [inline docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md:170]: Maybe we can come up with a less broad heading, like `Video Aspect Ratio Notes/Limitations/...`?

File: docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md [hunk 2]
```diff
@@ -162,3 +162,19 @@ test('Close a specific window', async t => {
...
+
+When you launch TestCafe with [video recording](./screenshots-and-videos.md) enabled, the resulting video scales to the initial (parent) browser window.
```
Comment (VasilyStrelyaev) [inline docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md:172]: Sorry if I sound nerdy, but there's no resulting video when you launch TestCafe... yet.

It's rather "recording starts inside (is framed to) the initial browser window"

File: docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md [hunk 3]
```diff
@@ -162,3 +162,19 @@ test('Close a specific window', async t => {
...
+
+When a child window opens, TestCafe continues recording in that window. If the child windows' size differs, the segments of the video that are recorded in these windows are resized to fit the aspect ratio of the main browser window. That part of the recording appears stretched as a result.
```
Comment (VasilyStrelyaev) [inline docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md:174]: When we say "segment" about an image, we mean the geometrical sense, i.e. a limited area.
When I saw "segments of the video", I thought you also mean "part of the whole recorded area".

I may be wrong, but maybe you'd better check the meaning of "segment" related to videos with Dirk?

**Review by VasilyStrelyaev — CHANGES_REQUESTED** [review]

File: docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md (hunk 3 above)
Comment (DIRECTcut) [reply docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md:174]: I agree, this is ambiguous.  Maybe `portion` is better

**Review by AlexKamaev — APPROVED** [review]

**Review by VasilyStrelyaev — APPROVED** [review]

File: docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md [hunk 4]
```diff
@@ -162,3 +162,19 @@ test('Close a specific window', async t => {
...
+
+## Limitations
+
+This section describes the limitations of multiple browser windows testing in TestCafe.
+
+### Recorded Video Aspect Ratio in Multiple Windows
+
+When you launch TestCafe with [video recording](./screenshots-and-videos.md) enabled, the recording scales to the initial (parent) browser window.
+
+When a child window opens, TestCafe continues recording in that window. If the child windows' size differs, the parts of the video that are recorded in these windows are resized to fit the aspect ratio of the main browser window. Portions of the recording appear stretched as a result.
```
Comment (dirk-pieterse) [inline docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md:174]:
```suggestion
When a child window opens, TestCafe continues recording in that window. If the child windows' size differs, the parts of the video that are recorded in these windows are resized to fit the aspect ratio of the main browser window. As a result, portions of the recording appear stretched.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**